Add readmes to reflection (#76567)
dotnet/runtime: CONTRIBUTING.md

Project maintainers will merge changes that improve the product significantly and broadly align with the [.NET Roadmap](https://github.com/dotnet/core/blob/master/roadmap.md).

> Maintainers will not merge changes that have narrowly-defined benefits, due to compatibility risk. The .NET codebase is used by several Microsoft products (for example, ASP.NET Core, .NET Framework 4.x, Windows Universal Apps) to enable execution of managed code. Other companies are building products on top of .NET, too. We may revert changes if they are found to be breaking.

Reviewer: ```suggestion
Maintainers will not merge changes that have narrowly-defined benefits, due to compatibility risk. .NET is used by several Microsoft products (for example, ASP.NET Core, .NET Framework 4.x, Windows Universal Apps) to enable execution of managed code. Other companies are building products on top of .NET, too. We may revert changes if they are found to be breaking.
```

* **DON'T** surprise us with big pull requests. Instead, file an issue and start
  a discussion so we can agree on a direction before you invest a large amount
  of time.

> * **DON'T** commit code that you didn't write. If you find code that you think is a good fit to add to .NET, file an issue and start a discussion before proceeding.

Author: Thanks but added "the":
```diff
- to add to .NET runtime,
+ to add to the .NET runtime,
```

### File Headers

> The following file header is the used for .NET. Please use it for new files.

Reviewer: ```suggestion
The following file header is the used for files in this repo. Please use it for new files.
```

#### Copying Files from Other Projects

> .NET uses some files from other projects, typically where a binary distribution does not exist or would be inconvenient.

Reviewer: ```suggestion
.NET runtime uses some files from other projects, typically where a binary distribution does not exist or would be inconvenient.
```

Author: Similar to above adding "the" makes it read better IMO:

```diff
- .NET runtime
+ The .NET runtime
```
